app/src/main/java/org/simple/clinic/medicalhistory/CholesterolReading.kt

```diff
@@ -0,0 +1,7 @@
+package org.simple.clinic.medicalhistory
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+data class CholesterolReading(val value: String) : Parcelable
```

Comment:
How about representing this as a float value instead? Since `String` can be anything instead of a numerical value.

Reply:
I have removed the cholesterol from the code as its not part of the currents scope, maybe we can do it once we start working on phase 3